binfmt/binfmt.h

```diff
@@ -117,6 +117,50 @@ void binfmt_freeargv(FAR char * const *argv);
 #  define binfmt_freeargv(argv)
 #endif
 
+/****************************************************************************
+ * Name: binfmt_copyenv
+ *
+ * Description:
+ *   In the kernel build, the environment exists in the parent's address
+ *   environment and, hence, will be inaccessible when we switch to the
+ *   address environment of the new process. So we do not have any real
+ *   option other than to copy the parents envp list into an intermediate
+ *   buffer that resides in neutral kernel memory.
+ *
+ * Input Parameters:
+ *   envp     - Allocated environment strings
+ *
+ * Returned Value:
+ *   A non-zero copy is returned on success.
+ *
+ ****************************************************************************/
+
+#ifndef CONFIG_DISABLE_ENVIRON
+FAR char * const *binfmt_copyenv(FAR char * const *envp);
```

Comment:
since env has the same layout as argv with #6083, we can share binfmt_copyargv now.

Reply:
Yes, and name can be changed to something more generic like copystrings or something

Reply:
Done